apps/data-types.md

```diff
@@ -479,6 +479,8 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 | minStocklevel | number | Indicates the minimum stock level of the product. Available if the field *Minimum stock level* is used in the administration area of a shop. Only available with the `products_write` authorisation. |
 | links | array of [link](page:apps-data-types#link) | The links to the product and product category. |
 | productVariationType | string | The type of the variation product. Can be one of *master*, *regular* or *variation*. |
+| productVariationMasterId | string | The unique identifier of the master product of a variation. Null if the product itself is a master or a regular product. |
```

Comment:
I'm not so happy with the naming of the attribute. Would  it be possible to change it to `variationProductId`? I don't want to use "master" so prominently, as what we're talking of here is the variation product, and the children are variations.

The unique identifier of the variation product. Null if the `productVariationType` is *master* or *regular*.

Reply:
Mhm, I understand your concern. The problem is, that it is not that easy to change the wording here because the code changes have been already integrated in epagesJ. To do so, we would need a new JIRA issue to change the naming of the attribute there as well as in epages6 and epages-ui. As there has not been any suggestion, I sticked to the wording that is used in epages-ui internally and used it here too ...